published/c-c++/functional-style-programming-using-c/article.md

```diff
@@ -1,17 +1,17 @@
-> LEARNING PROGRAMMING AND LEARNING PROGRAMMING LANGUAGE ARE NOT THE SAME. MOST OF OUR TEXTBOOKS TEACH US PROGRAMMING LANGUAGES. IN TRUTH, A PROGRAMMING LANGUAGE IS JUST A TOOL THAT WE USE TO IMPLEMENT AN EXECUTABLE PROGRAM. WE NEED RESOURCES THAT TEACH US HOW TO _SOLVE PROBLEMS_ USING A PROGRAMMING LANGUAGE.
+> Learning programming and learning programming language are not the same. most of our textbooks teach us programming languages. In truth, a programming language is just a tool that we use to implement an executable program. We need resources that teach us how to _solve problems_ using a programming language.
```

Comment:
_solve problems_ 
can be substituted by 
"solve problems" 
or simply
solve problems